_data/members.yml

```diff
@@ -710,7 +710,12 @@ tomasz.szewczyk:
 piotr.prusinski:
     name: Piotr Prusiński
     bio: I’m a Software engineer who likes to develop in every direction. I’m a fan of treating soft skills on par with hard skills.
-
+    
```

Comment:
To tylko do usunięcia, reszta według mnie gites 👍

Reply:
Usunęłam, resolved!